backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/BpkButton.kt

```diff
@@ -133,6 +135,66 @@ fun BpkButton(
   }
 }
 
+@Composable
+fun BpkButton(
+  @DrawableRes iconResource: Int,
```

Comment:
rather than fixing this to the `iconResource` (and therefore creating a dependency on the existing resource system) could this be an `icon` composable instead?

Reply:
changed to use `Painter` instead